data/human/free worlds 1 start.txt

```diff
@@ -792,7 +792,7 @@ mission "FW Plasma Testing 1B"
 				`	"Well, we try to only blow things up when strictly necessary."`
 			
 			label explain
-			`	"Of course," he says. "Anyway, I bet you're wondering what new toys we're going to let you play with." You nod. He says, "We've been trying to come up with a weapon for our capital ships, something that's a match for one of the Navy's heavy laser turrets. Something more elegant than those awful quad blaster turrets the Syndicate makes. So I thought, why not take the biggest thing we've got, and then mount a pair of them on a turret?" He beckons for you to follow, and leads you through a door and into a hangar.`
+			`	"Of course," he says. "Anyway, I bet you're wondering what new toys we're going to let you play with." You nod. He says, "We've been trying to come up with a weapon for our capital ships, something that's a match for one of the Navy's heavy laser turrets. Something more elegant than those awful quad Blaster Turrets the Syndicate makes. So I thought, why not take the biggest thing we've got, and then mount a pair of them on a turret?" He beckons for you to follow, and leads you through a door and into a hangar.`
```

Comment:
```suggestion
			`	"Of course," he says. "Anyway, I bet you're wondering what new toys we're going to let you play with." You nod. He says, "We've been trying to come up with a weapon for our capital ships, something that's a match for one of the Navy's heavy laser turrets. Something more elegant than those awful quad blaster turrets the Syndicate makes. So I thought, why not take the biggest thing we've got, and then mount a pair of them on a turret?" He beckons for you to follow, and leads you through a door and into a hangar.`
```
This is getting fixed in Stage 2, so just in case this gets merged afterwards and causes a change we don't want to see.

Reply:
Well, if this is changed, then `Heavy Laser Turrets` in the same line would be changed to `heavy laser turrets` if Stage 2 is merged first. This suggestion is just housekeeping since I figured it was an oversight.

I'm not doing it because I think it should be `quad blaster turrets`.